backend/apps/core/management/commands/seed_companies.py

```
@@ -0,0 +1,467 @@
+                
+                break  # Successfully read, exit loop
+            except UnicodeDecodeError:
+                continue  # Try next encoding
+            except Exception as e:
+                if encoding == encodings[-1]:  # Last encoding
+                    raise Exception(f"Error reading CSV file: {str(e)}")
+                continue
+        
+        return companies
+
+    def handle(self, *args, **options):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)